src/php/ext/grpc/channel.c

```diff
@@ -586,6 +586,13 @@ void php_grpc_delete_persistent_list_entry(char *key, php_grpc_int key_len
   gpr_mu_unlock(&global_persistent_list_mu);
 }
 
+// Clean all channels in the persistent list
+// Called at post fork
+void php_grpc_clean_persistent_list(TSRMLS_D) {
```

Comment:
Instead of refactoring out this `php_grpc_clean_persistent_list` function in `channel.c`, perhaps we can limit our changes to only `php_grpc.c`, and refactor out these 4 lines instead https://github.com/grpc/grpc/blob/master/src/php/ext/grpc/php_grpc.c#L435-L438, and call them in those 2 separate places, but within `php_grpc.c`

Reply:
comment addressed